rhp/v2/sectorsizerelease.go

```diff
@@ -0,0 +1,14 @@
+//go:build !testing
```

Comment:
are we forced to do this because we're testing against the siad host?

Reply:
Yes, the upload will fail otherwise because renterd pads the slab to 4MiB while the host wants 4KiB sectors.